db/database.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+
+SQLALCHEMY_DATABASE_URL = "sqlite:///./library.sqlite3"

```

Comment:
The database URL is set for SQLite. Ensure this is suitable for your production environment, as SQLite is typically used for development and testing.